!SESSION 2023-04-16 18:34:58.825 -----------------------------------------------
eclipse.buildId=4.27.0.I20230302-0300
java.version=17.0.4.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_CA
Framework arguments:  -keyring /Users/manveer/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -keyring /Users/manveer/.eclipse_keyring

This is a continuation of log file /Users/manveer/Documents/Eclipse/CP164/.metadata/.bak_1.log
Created Time: 2023-04-17 14:16:14.661

!ENTRY org.python.pydev.shared_core 4 4 2023-04-17 14:16:14.664
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-17 14:16:14.666
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-17 14:16:14.666
!MESSAGE Unable to find mypy. Project: sidh9846_e
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_e
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-17 14:16:58.017
!MESSAGE /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/third_party/pep8/pycodestyle.py:113: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')

!STACK 0
java.lang.RuntimeException: /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/third_party/pep8/pycodestyle.py:113: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')

	at org.python.pydev.core.log.Log.log(Log.java:33)
	at org.python.pydev.core.pep8.Pep8Runner.runWithPep8BaseScript(Pep8Runner.java:106)
	at org.python.pydev.ast.formatter.PyFormatter.formatStrAutopep8OrPyDev(PyFormatter.java:918)
	at org.python.pydev.ast.formatter.PyFormatter.formatStrAutopep8OrPyDev(PyFormatter.java:971)
	at org.python.pydev.ast.formatter.PyFormatter.formatStrAutopep8OrPyDev(PyFormatter.java:961)
	at org.python.pydev.ast.formatter.PyFormatter.formatAll(PyFormatter.java:995)
	at org.python.pydev.editor.actions.PyFormatAction.formatAll(PyFormatAction.java:153)
	at org.python.pydev.editor.actions.PyFormatAction.applyFormatAction(PyFormatAction.java:102)
	at org.python.pydev.editor.PyEdit.performSave(PyEdit.java:885)
	at org.eclipse.ui.texteditor.AbstractTextEditor.doSave(AbstractTextEditor.java:4805)
	at org.python.pydev.editor.PyEdit.doSave(PyEdit.java:980)
	at org.eclipse.ui.texteditor.AbstractTextEditor$TextEditorSavable.doSave(AbstractTextEditor.java:7185)
	at org.eclipse.ui.Saveable.doSave(Saveable.java:213)
	at org.eclipse.ui.internal.SaveableHelper.doSaveModel(SaveableHelper.java:333)
	at org.eclipse.ui.internal.SaveableHelper.lambda$1(SaveableHelper.java:196)
	at org.eclipse.ui.internal.SaveableHelper.lambda$3(SaveableHelper.java:271)
	at org.eclipse.jface.operation.ModalContext.runInCurrentThread(ModalContext.java:434)
	at org.eclipse.jface.operation.ModalContext.run(ModalContext.java:352)
	at org.eclipse.ui.internal.WorkbenchWindow.lambda$7(WorkbenchWindow.java:2346)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:67)
	at org.eclipse.ui.internal.WorkbenchWindow.run(WorkbenchWindow.java:2344)
	at org.eclipse.ui.internal.SaveableHelper.runProgressMonitorOperation(SaveableHelper.java:278)
	at org.eclipse.ui.internal.SaveableHelper.runProgressMonitorOperation(SaveableHelper.java:260)
	at org.eclipse.ui.internal.SaveableHelper.saveModels(SaveableHelper.java:207)
	at org.eclipse.ui.internal.SaveableHelper.savePart(SaveableHelper.java:150)
	at org.eclipse.ui.internal.WorkbenchPage.saveSaveable(WorkbenchPage.java:3802)
	at org.eclipse.ui.internal.WorkbenchPage.saveEditor(WorkbenchPage.java:3815)
	at org.eclipse.ui.internal.handlers.SaveHandler.execute(SaveHandler.java:82)
	at org.eclipse.ui.internal.handlers.HandlerProxy.execute(HandlerProxy.java:283)
	at org.eclipse.ui.internal.handlers.E4HandlerProxy.execute(E4HandlerProxy.java:97)
	at jdk.internal.reflect.GeneratedMethodAccessor59.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:58)
	at org.eclipse.e4.core.internal.di.InjectorImpl.invokeUsingClass(InjectorImpl.java:317)
	at org.eclipse.e4.core.internal.di.InjectorImpl.invoke(InjectorImpl.java:251)
	at org.eclipse.e4.core.contexts.ContextInjectionFactory.invoke(ContextInjectionFactory.java:173)
	at org.eclipse.e4.core.commands.internal.HandlerServiceHandler.execute(HandlerServiceHandler.java:156)
	at org.eclipse.core.commands.Command.executeWithChecks(Command.java:488)
	at org.eclipse.core.commands.ParameterizedCommand.executeWithChecks(ParameterizedCommand.java:485)
	at org.eclipse.e4.core.commands.internal.HandlerServiceImpl.executeHandler(HandlerServiceImpl.java:213)
	at org.eclipse.e4.ui.bindings.keys.KeyBindingDispatcher.executeCommand(KeyBindingDispatcher.java:308)
	at org.eclipse.e4.ui.bindings.keys.KeyBindingDispatcher.press(KeyBindingDispatcher.java:580)
	at org.eclipse.e4.ui.bindings.keys.KeyBindingDispatcher.processKeyEvent(KeyBindingDispatcher.java:647)
	at org.eclipse.e4.ui.bindings.keys.KeyBindingDispatcher.filterKeySequenceBindings(KeyBindingDispatcher.java:439)
	at org.eclipse.e4.ui.bindings.keys.KeyBindingDispatcher$KeyDownFilter.handleEvent(KeyBindingDispatcher.java:96)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.filterEvent(Display.java:1217)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4641)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1524)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1547)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1532)
	at org.eclipse.swt.widgets.Widget.sendKeyEvent(Widget.java:1561)
	at org.eclipse.swt.widgets.Widget.sendKeyEvent(Widget.java:1557)
	at org.eclipse.swt.widgets.Canvas.sendKeyEvent(Canvas.java:522)
	at org.eclipse.swt.widgets.Control.doCommandBySelector(Control.java:1085)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6492)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSend(Native Method)
	at org.eclipse.swt.internal.cocoa.NSResponder.interpretKeyEvents(NSResponder.java:59)
	at org.eclipse.swt.widgets.Composite.keyDown(Composite.java:607)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6324)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSendSuper(Native Method)
	at org.eclipse.swt.widgets.Widget.callSuper(Widget.java:236)
	at org.eclipse.swt.widgets.Widget.windowSendEvent(Widget.java:2264)
	at org.eclipse.swt.widgets.Shell.windowSendEvent(Shell.java:2511)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6444)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSendSuper(Native Method)
	at org.eclipse.swt.widgets.Display.applicationSendEvent(Display.java:5692)
	at org.eclipse.swt.widgets.Display.applicationProc(Display.java:5831)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSend(Native Method)
	at org.eclipse.swt.internal.cocoa.NSApplication.sendEvent(NSApplication.java:117)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3986)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:643)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:550)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1467)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-17 14:16:58.109
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-17 14:16:58.109
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-17 14:16:58.110
!MESSAGE Unable to find mypy. Project: sidh9846_e
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_e
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
!SESSION 2023-04-25 21:41:27.496 -----------------------------------------------
eclipse.buildId=4.27.0.I20230302-0300
java.version=17.0.4.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_CA
Framework arguments:  -keyring /Users/manveer/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -keyring /Users/manveer/.eclipse_keyring

!ENTRY org.eclipse.jface 2 0 2023-04-25 21:41:40.410
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2023-04-25 21:41:40.410
!MESSAGE A conflict occurred for ALT+COMMAND+R:
Binding(ALT+COMMAND+R,
	ParameterizedCommand(Command(org.python.pydev.debug.setnext,Set Next Statement,
		,
		Category(org.python.pydev.ui.category.run,PyDev - Run,Python run category,true),
		WorkbenchHandlerServiceHandler("org.python.pydev.debug.setnext"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
Binding(ALT+COMMAND+R,
	ParameterizedCommand(Command(org.eclipse.jdt.ui.edit.text.java.rename.element,Rename - Refactoring ,
		Rename the selected element,
		Category(org.eclipse.jdt.ui.category.refactoring,Refactor - Java,Java Refactoring Actions,true),
		WorkbenchHandlerServiceHandler("org.eclipse.jdt.ui.edit.text.java.rename.element"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,cocoa,system)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:41:46.181
!MESSAGE Error. returning: 0
!STACK 0
java.nio.file.NoSuchFileException: /Users/manveer/Documents/Eclipse/CP164/FinalPractice_DataStructures/src/BST_linked_test.py
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:148)
	at java.base/java.nio.file.Files.readAttributes(Files.java:1851)
	at java.base/java.nio.file.Files.getLastModifiedTime(Files.java:2402)
	at org.python.pydev.shared_core.io.FileUtils.lastModified(FileUtils.java:1081)
	at org.python.pydev.shared_core.io.FileUtils.lastModified(FileUtils.java:1072)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalDependencyInfo.addInfoToModuleOnRestoreInsertCommand(AbstractAdditionalDependencyInfo.java:547)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.processInsert(AbstractAdditionalInfoWithBuild.java:192)
	at org.python.pydev.core.DeltaSaver$DeltaInsertCommand.processWith(DeltaSaver.java:98)
	at org.python.pydev.core.DeltaSaver.processDeltas(DeltaSaver.java:318)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreSavedInfo(AbstractAdditionalInfoWithBuild.java:103)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalDependencyInfo.loadContentsFromFile(AbstractAdditionalDependencyInfo.java:517)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalDependencyInfo.load(AbstractAdditionalDependencyInfo.java:432)
	at com.python.pydev.analysis.additionalinfo.AdditionalProjectInterpreterInfo.getAdditionalInfoForProject(AdditionalProjectInterpreterInfo.java:249)
	at com.python.pydev.analysis.additionalinfo.AdditionalProjectInterpreterInfo.recreateAllInfo(AdditionalProjectInterpreterInfo.java:339)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyProjectPythonpathRestored(InterpreterObserver.java:41)
	at org.python.pydev.plugin.nature.PythonNature$RebuildPythonNatureModules.run(PythonNature.java:201)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:41:46.336
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:41:46.339
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:41:46.340
!MESSAGE Unable to find mypy. Project: sidh9846_e
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_e
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:42:12.025
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:42:12.034
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:42:12.036
!MESSAGE Unable to find mypy. Project: sidh9846_e
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_e
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:42:18.506
!MESSAGE Unable to find mypy. Project: sidh9846_data_structures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_data_structures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:42:46.603
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:42:46.611
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-04-25 21:42:46.611
!MESSAGE Unable to find mypy. Project: FinalPractice_DataStructures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: FinalPractice_DataStructures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
!SESSION 2023-10-30 20:18:40.910 -----------------------------------------------
eclipse.buildId=4.29.0.I20230903-1000
java.version=17.0.4.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_CA
Framework arguments:  -keyring /Users/manveer/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -keyring /Users/manveer/.eclipse_keyring

!ENTRY org.eclipse.core.resources 4 567 2023-10-30 20:18:56.448
!MESSAGE Workspace restored, but some problems occurred.
!SUBENTRY 1 org.eclipse.core.resources 4 567 2023-10-30 20:18:56.448
!MESSAGE Could not read metadata for 'sidh9846_e'.
!STACK 1
org.eclipse.core.internal.resources.ResourceException(/sidh9846_e)[567]: java.lang.Exception: The project description file (.project) for 'sidh9846_e' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
	at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:955)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:954)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:934)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:790)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1616)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:2620)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:2324)
	at org.eclipse.core.resources.ResourcesPlugin$WorkspaceInitCustomizer.addingService(ResourcesPlugin.java:571)
	at org.eclipse.core.resources.ResourcesPlugin$WorkspaceInitCustomizer.addingService(ResourcesPlugin.java:1)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:1)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:321)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:264)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:545)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:818)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:810)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:767)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:1032)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:371)
	at org.eclipse.osgi.container.Module.doStart(Module.java:605)
	at org.eclipse.osgi.container.Module.start(Module.java:468)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:513)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:117)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:570)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:335)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:397)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:41)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass0(BundleLoader.java:496)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:416)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:168)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:153)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)
!SUBENTRY 2 org.eclipse.core.resources 4 567 2023-10-30 20:18:56.452
!MESSAGE The project description file (.project) for 'sidh9846_e' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
!STACK 0
java.lang.Exception: The project description file (.project) for 'sidh9846_e' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
	at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:955)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:954)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:934)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:790)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1616)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:2620)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:2324)
	at org.eclipse.core.resources.ResourcesPlugin$WorkspaceInitCustomizer.addingService(ResourcesPlugin.java:571)
	at org.eclipse.core.resources.ResourcesPlugin$WorkspaceInitCustomizer.addingService(ResourcesPlugin.java:1)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:1)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:321)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:264)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:545)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:818)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:810)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:767)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:1032)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:371)
	at org.eclipse.osgi.container.Module.doStart(Module.java:605)
	at org.eclipse.osgi.container.Module.start(Module.java:468)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:513)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:117)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:570)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:335)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:397)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:41)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass0(BundleLoader.java:496)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:416)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:168)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:153)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)
!SUBENTRY 1 org.eclipse.core.resources 4 567 2023-10-30 20:18:56.453
!MESSAGE Could not read metadata for 'sidh9846_m'.
!STACK 1
org.eclipse.core.internal.resources.ResourceException(/sidh9846_m)[567]: java.lang.Exception: The project description file (.project) for 'sidh9846_m' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
	at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:955)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:954)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:934)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:790)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1616)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:2620)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:2324)
	at org.eclipse.core.resources.ResourcesPlugin$WorkspaceInitCustomizer.addingService(ResourcesPlugin.java:571)
	at org.eclipse.core.resources.ResourcesPlugin$WorkspaceInitCustomizer.addingService(ResourcesPlugin.java:1)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:1)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:321)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:264)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:545)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:818)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:810)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:767)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:1032)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:371)
	at org.eclipse.osgi.container.Module.doStart(Module.java:605)
	at org.eclipse.osgi.container.Module.start(Module.java:468)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:513)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:117)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:570)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:335)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:397)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:41)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass0(BundleLoader.java:496)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:416)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:168)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:153)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)
!SUBENTRY 2 org.eclipse.core.resources 4 567 2023-10-30 20:18:56.453
!MESSAGE The project description file (.project) for 'sidh9846_m' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
!STACK 0
java.lang.Exception: The project description file (.project) for 'sidh9846_m' is missing.  This file contains important information about the project.  The project will not function properly until this file is restored.
	at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
	at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
	at org.eclipse.core.internal.localstore.FileSystemResourceManager.read(FileSystemResourceManager.java:955)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:954)
	at org.eclipse.core.internal.resources.SaveManager.restoreMetaInfo(SaveManager.java:934)
	at org.eclipse.core.internal.resources.SaveManager.restore(SaveManager.java:790)
	at org.eclipse.core.internal.resources.SaveManager.startup(SaveManager.java:1616)
	at org.eclipse.core.internal.resources.Workspace.startup(Workspace.java:2620)
	at org.eclipse.core.internal.resources.Workspace.open(Workspace.java:2324)
	at org.eclipse.core.resources.ResourcesPlugin$WorkspaceInitCustomizer.addingService(ResourcesPlugin.java:571)
	at org.eclipse.core.resources.ResourcesPlugin$WorkspaceInitCustomizer.addingService(ResourcesPlugin.java:1)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:1)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:321)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:264)
	at org.eclipse.core.resources.ResourcesPlugin.start(ResourcesPlugin.java:545)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:818)
	at org.eclipse.osgi.internal.framework.BundleContextImpl$2.run(BundleContextImpl.java:1)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:810)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:767)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:1032)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:371)
	at org.eclipse.osgi.container.Module.doStart(Module.java:605)
	at org.eclipse.osgi.container.Module.start(Module.java:468)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:513)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:117)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:570)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:335)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:397)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:41)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass0(BundleLoader.java:496)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:416)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:168)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:153)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)

!ENTRY org.eclipse.jface 2 0 2023-10-30 20:18:59.498
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2023-10-30 20:18:59.498
!MESSAGE A conflict occurred for ALT+COMMAND+R:
Binding(ALT+COMMAND+R,
	ParameterizedCommand(Command(org.python.pydev.debug.setnext,Set Next Statement,
		,
		Category(org.python.pydev.ui.category.run,PyDev - Run,Python run category,true),
		WorkbenchHandlerServiceHandler("org.python.pydev.debug.setnext"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
Binding(ALT+COMMAND+R,
	ParameterizedCommand(Command(org.eclipse.jdt.ui.edit.text.java.rename.element,Rename - Refactoring ,
		Rename the selected element,
		Category(org.eclipse.jdt.ui.category.refactoring,Refactor - Java,Java Refactoring Actions,true),
		WorkbenchHandlerServiceHandler("org.eclipse.jdt.ui.edit.text.java.rename.element"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,cocoa,system)

!ENTRY org.eclipse.e4.ui.workbench 2 0 2023-10-30 20:19:02.431
!MESSAGE Removing PartDescriptorImpl with the "org.eclipse.ui.views.ResourceNavigator" id and the "Navigator (Deprecated)" label.It points to the non available "bundleclass://org.eclipse.ui.workbench/org.eclipse.ui.internal.e4.compatibility.CompatibilityView" class. Bundle might have been uninstalled

!ENTRY org.python.pydev.shared_core 1 1 2023-10-30 20:19:02.500
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 1 1 2023-10-30 20:19:02.523
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 1 1 2023-10-30 20:19:02.524
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:19:23.385
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:19:23.391
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:19:23.392
!MESSAGE Unable to find mypy. Project: sidh9846_data_structures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_data_structures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:19:23.672
!MESSAGE Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 58406

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

!STACK 0
java.lang.RuntimeException: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 58406

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.core.log.Log.log(Log.java:33)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:287)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.editor.PyEdit$2.run(PyEdit.java:560)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:19:32.591
!MESSAGE No output was in the standard output when
trying to create the interpreter info for: /usr/bin/python3
The error output contains:>>xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
<<
!STACK 0
java.lang.RuntimeException: No output was in the standard output when
trying to create the interpreter info for: /usr/bin/python3
The error output contains:>>xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
<<
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.createInfoFromOutput(AbstractInterpreterManager.java:369)
	at org.python.pydev.ast.interpreter_managers.PythonInterpreterManager.doCreateInterpreterInfo(PythonInterpreterManager.java:70)
	at org.python.pydev.ast.interpreter_managers.PythonInterpreterManager.internalCreateInterpreterInfo(PythonInterpreterManager.java:43)
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.createInterpreterInfo(AbstractInterpreterManager.java:329)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManager$CreateInterpreterInfoCallback.createInterpreterInfo(SyncSystemModulesManager.java:172)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManager.updateStructures(SyncSystemModulesManager.java:278)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler$SynchJob.run(SyncSystemModulesManagerScheduler.java:234)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:47:35.962
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:47:35.969
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:47:35.969
!MESSAGE Unable to find mypy. Project: sidh9846_data_structures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_data_structures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 20:47:36.024
!MESSAGE Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 60117

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

!STACK 0
java.lang.RuntimeException: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 60117

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.core.log.Log.log(Log.java:33)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:287)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.editor.PyEdit$2.run(PyEdit.java:560)

!ENTRY org.eclipse.ui.ide 4 4 2023-10-30 20:47:36.457
!MESSAGE Not properly disposed SWT resource
!STACK 0
java.lang.Error: SWT Resource was not properly disposed
	at org.eclipse.swt.graphics.Resource.initNonDisposeTracking(Resource.java:172)
	at org.eclipse.swt.graphics.Resource.<init>(Resource.java:120)
	at org.eclipse.swt.graphics.Font.<init>(Font.java:100)
	at org.eclipse.ui.texteditor.AbstractTextEditor$6.gesture(AbstractTextEditor.java:3858)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:152)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4646)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1524)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1547)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1532)
	at org.eclipse.swt.widgets.Control.gestureEvent(Control.java:1550)
	at org.eclipse.swt.widgets.Control.magnifyWithEvent(Control.java:2508)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6654)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSendSuper(Native Method)
	at org.eclipse.swt.widgets.Widget.callSuper(Widget.java:236)
	at org.eclipse.swt.widgets.Widget.windowSendEvent(Widget.java:2264)
	at org.eclipse.swt.widgets.Shell.windowSendEvent(Shell.java:2511)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6444)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSendSuper(Native Method)
	at org.eclipse.swt.widgets.Display.applicationSendEvent(Display.java:5692)
	at org.eclipse.swt.widgets.Display.applicationProc(Display.java:5831)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSend(Native Method)
	at org.eclipse.swt.internal.cocoa.NSApplication.sendEvent(NSApplication.java:117)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3986)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:645)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:552)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)

!ENTRY org.eclipse.ui.ide 4 4 2023-10-30 20:47:36.467
!MESSAGE Not properly disposed SWT resource
!STACK 0
java.lang.Error: SWT Resource was not properly disposed
	at org.eclipse.swt.graphics.Resource.initNonDisposeTracking(Resource.java:172)
	at org.eclipse.swt.graphics.Resource.<init>(Resource.java:120)
	at org.eclipse.swt.graphics.Font.<init>(Font.java:100)
	at org.eclipse.ui.texteditor.AbstractTextEditor$6.gesture(AbstractTextEditor.java:3858)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:152)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4646)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1524)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1547)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1532)
	at org.eclipse.swt.widgets.Control.gestureEvent(Control.java:1550)
	at org.eclipse.swt.widgets.Control.magnifyWithEvent(Control.java:2508)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6654)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSendSuper(Native Method)
	at org.eclipse.swt.widgets.Widget.callSuper(Widget.java:236)
	at org.eclipse.swt.widgets.Widget.windowSendEvent(Widget.java:2264)
	at org.eclipse.swt.widgets.Shell.windowSendEvent(Shell.java:2511)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6444)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSendSuper(Native Method)
	at org.eclipse.swt.widgets.Display.applicationSendEvent(Display.java:5692)
	at org.eclipse.swt.widgets.Display.applicationProc(Display.java:5831)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSend(Native Method)
	at org.eclipse.swt.internal.cocoa.NSApplication.sendEvent(NSApplication.java:117)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3986)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:645)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:552)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 21:08:13.384
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 21:08:13.392
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 21:08:13.392
!MESSAGE Unable to find mypy. Project: sidh9846_data_structures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_data_structures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 21:08:13.575
!MESSAGE Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 61380

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

!STACK 0
java.lang.RuntimeException: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 61380

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.core.log.Log.log(Log.java:33)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:287)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.editor.PyEdit$2.run(PyEdit.java:560)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 23:07:06.867
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 23:07:06.870
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 23:07:06.870
!MESSAGE Unable to find mypy. Project: sidh9846_data_structures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_data_structures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-30 23:07:06.913
!MESSAGE Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 49769

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

!STACK 0
java.lang.RuntimeException: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 49769

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.core.log.Log.log(Log.java:33)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:287)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.editor.PyEdit$2.run(PyEdit.java:560)

!ENTRY org.eclipse.ui.ide 4 4 2023-10-31 16:44:04.973
!MESSAGE Not properly disposed SWT resource
!STACK 0
java.lang.Error: SWT Resource was not properly disposed
	at org.eclipse.swt.graphics.Resource.initNonDisposeTracking(Resource.java:172)
	at org.eclipse.swt.graphics.Resource.<init>(Resource.java:120)
	at org.eclipse.swt.graphics.Font.<init>(Font.java:100)
	at org.eclipse.ui.texteditor.AbstractTextEditor$6.gesture(AbstractTextEditor.java:3858)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:152)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4646)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1524)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1547)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1532)
	at org.eclipse.swt.widgets.Control.gestureEvent(Control.java:1550)
	at org.eclipse.swt.widgets.Control.magnifyWithEvent(Control.java:2508)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6654)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSendSuper(Native Method)
	at org.eclipse.swt.widgets.Widget.callSuper(Widget.java:236)
	at org.eclipse.swt.widgets.Widget.windowSendEvent(Widget.java:2264)
	at org.eclipse.swt.widgets.Shell.windowSendEvent(Shell.java:2511)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6444)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSendSuper(Native Method)
	at org.eclipse.swt.widgets.Display.applicationSendEvent(Display.java:5692)
	at org.eclipse.swt.widgets.Display.applicationProc(Display.java:5831)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSend(Native Method)
	at org.eclipse.swt.internal.cocoa.NSApplication.sendEvent(NSApplication.java:117)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3986)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:645)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:552)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-31 18:32:20.466
!MESSAGE Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 56383

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

!STACK 0
java.lang.RuntimeException: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 56383

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.core.log.Log.log(Log.java:33)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:287)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.ast.codecompletion.revisited.modules.CompiledModule.createInnerFromServer(CompiledModule.java:345)
	at org.python.pydev.ast.codecompletion.revisited.modules.CompiledModule.getGlobalTokens(CompiledModule.java:551)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.internalGenerateGetCompletionsForModule(AbstractASTManager.java:890)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:710)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:660)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.internalGenerateGetCompletionsForModule(AbstractASTManager.java:902)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:710)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:670)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.internalFindGlobalTokDef(SourceModule.java:1370)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findGlobalTokDef(SourceModule.java:1338)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinitionsFromModAndTok(SourceModule.java:1280)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:1262)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:825)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:1)
	at org.python.pydev.ast.refactoring.PyRefactoringFindDefinition.findActualDefinition(PyRefactoringFindDefinition.java:211)
	at org.python.pydev.ast.refactoring.PyRefactoringFindDefinition.findActualDefinition(PyRefactoringFindDefinition.java:92)
	at org.python.pydev.editor.hover.PyDocstringTextHover.getDocstringHover(PyDocstringTextHover.java:101)
	at org.python.pydev.editor.hover.PyDocstringTextHover.getHoverInfo(PyDocstringTextHover.java:59)
	at org.python.pydev.editor.hover.PydevCombiningHover.getHoverInfo(PydevCombiningHover.java:141)
	at org.python.pydev.editor.hover.PyEditorTextHoverProxy.getHoverInfo(PyEditorTextHoverProxy.java:59)
	at org.eclipse.jface.text.TextViewerHoverManager$1.run(TextViewerHoverManager.java:157)

!ENTRY org.python.pydev.shared_core 4 4 2023-10-31 18:32:20.473
!MESSAGE Error while getting info for module:builtins. Project: P/sidh9846_data_structures
!STACK 1
org.eclipse.core.runtime.CoreException: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 56383

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:288)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.ast.codecompletion.revisited.modules.CompiledModule.createInnerFromServer(CompiledModule.java:345)
	at org.python.pydev.ast.codecompletion.revisited.modules.CompiledModule.getGlobalTokens(CompiledModule.java:551)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.internalGenerateGetCompletionsForModule(AbstractASTManager.java:890)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:710)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:660)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.internalGenerateGetCompletionsForModule(AbstractASTManager.java:902)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:710)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:670)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.internalFindGlobalTokDef(SourceModule.java:1370)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findGlobalTokDef(SourceModule.java:1338)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinitionsFromModAndTok(SourceModule.java:1280)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:1262)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:825)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:1)
	at org.python.pydev.ast.refactoring.PyRefactoringFindDefinition.findActualDefinition(PyRefactoringFindDefinition.java:211)
	at org.python.pydev.ast.refactoring.PyRefactoringFindDefinition.findActualDefinition(PyRefactoringFindDefinition.java:92)
	at org.python.pydev.editor.hover.PyDocstringTextHover.getDocstringHover(PyDocstringTextHover.java:101)
	at org.python.pydev.editor.hover.PyDocstringTextHover.getHoverInfo(PyDocstringTextHover.java:59)
	at org.python.pydev.editor.hover.PydevCombiningHover.getHoverInfo(PydevCombiningHover.java:141)
	at org.python.pydev.editor.hover.PyEditorTextHoverProxy.getHoverInfo(PyEditorTextHoverProxy.java:59)
	at org.eclipse.jface.text.TextViewerHoverManager$1.run(TextViewerHoverManager.java:157)
Caused by: java.lang.Exception: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 56383

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	... 25 more
!SUBENTRY 1 org.python.pydev 4 4 2023-10-31 18:32:20.475
!MESSAGE Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 56383

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

!STACK 0
java.lang.Exception: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 56383

 - Environment:
APP_ICON_47669=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_47669=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.0pCqCwGK0x/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D21BF276-6E35-4E86-8A04-E4220ADEE697
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:288)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.ast.codecompletion.revisited.modules.CompiledModule.createInnerFromServer(CompiledModule.java:345)
	at org.python.pydev.ast.codecompletion.revisited.modules.CompiledModule.getGlobalTokens(CompiledModule.java:551)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.internalGenerateGetCompletionsForModule(AbstractASTManager.java:890)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:710)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:660)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.internalGenerateGetCompletionsForModule(AbstractASTManager.java:902)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:710)
	at org.python.pydev.ast.codecompletion.revisited.AbstractASTManager.getCompletionsForModule(AbstractASTManager.java:670)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.internalFindGlobalTokDef(SourceModule.java:1370)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findGlobalTokDef(SourceModule.java:1338)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinitionsFromModAndTok(SourceModule.java:1280)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:1262)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:825)
	at org.python.pydev.ast.codecompletion.revisited.modules.SourceModule.findDefinition(SourceModule.java:1)
	at org.python.pydev.ast.refactoring.PyRefactoringFindDefinition.findActualDefinition(PyRefactoringFindDefinition.java:211)
	at org.python.pydev.ast.refactoring.PyRefactoringFindDefinition.findActualDefinition(PyRefactoringFindDefinition.java:92)
	at org.python.pydev.editor.hover.PyDocstringTextHover.getDocstringHover(PyDocstringTextHover.java:101)
	at org.python.pydev.editor.hover.PyDocstringTextHover.getHoverInfo(PyDocstringTextHover.java:59)
	at org.python.pydev.editor.hover.PydevCombiningHover.getHoverInfo(PydevCombiningHover.java:141)
	at org.python.pydev.editor.hover.PyEditorTextHoverProxy.getHoverInfo(PyEditorTextHoverProxy.java:59)
	at org.eclipse.jface.text.TextViewerHoverManager$1.run(TextViewerHoverManager.java:157)
!SESSION 2023-11-18 17:40:19.442 -----------------------------------------------
eclipse.buildId=4.29.0.I20230903-1000
java.version=17.0.4.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_CA
Framework arguments:  -keyring /Users/manveer/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -keyring /Users/manveer/.eclipse_keyring

!ENTRY org.eclipse.jface 2 0 2023-11-18 17:40:30.986
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2023-11-18 17:40:30.986
!MESSAGE A conflict occurred for ALT+COMMAND+R:
Binding(ALT+COMMAND+R,
	ParameterizedCommand(Command(org.python.pydev.debug.setnext,Set Next Statement,
		,
		Category(org.python.pydev.ui.category.run,PyDev - Run,Python run category,true),
		WorkbenchHandlerServiceHandler("org.python.pydev.debug.setnext"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
Binding(ALT+COMMAND+R,
	ParameterizedCommand(Command(org.eclipse.jdt.ui.edit.text.java.rename.element,Rename - Refactoring ,
		Rename the selected element,
		Category(org.eclipse.jdt.ui.category.refactoring,Refactor - Java,Java Refactoring Actions,true),
		WorkbenchHandlerServiceHandler("org.eclipse.jdt.ui.edit.text.java.rename.element"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,cocoa,system)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:40:34.133
!MESSAGE Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 50467

 - Environment:
APP_ICON_90082=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_90082=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.ubAWpVqq9T/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D3EC79B6-0B6D-40C8-ACCD-FC04C4262A55
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

!STACK 0
java.lang.RuntimeException: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 50467

 - Environment:
APP_ICON_90082=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_90082=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.ubAWpVqq9T/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D3EC79B6-0B6D-40C8-ACCD-FC04C4262A55
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.core.log.Log.log(Log.java:33)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:287)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.editor.PyEdit$2.run(PyEdit.java:560)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:40:35.598
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 1 1 2023-11-18 17:40:35.600
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 1 1 2023-11-18 17:40:35.601
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:40:35.601
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 1 1 2023-11-18 17:40:35.602
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:40:35.603
!MESSAGE Unable to find mypy. Project: sidh9846_data_structures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_data_structures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:41:05.682
!MESSAGE No output was in the standard output when
trying to create the interpreter info for: /usr/bin/python3
The error output contains:>>xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
<<
!STACK 0
java.lang.RuntimeException: No output was in the standard output when
trying to create the interpreter info for: /usr/bin/python3
The error output contains:>>xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
<<
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.createInfoFromOutput(AbstractInterpreterManager.java:369)
	at org.python.pydev.ast.interpreter_managers.PythonInterpreterManager.doCreateInterpreterInfo(PythonInterpreterManager.java:70)
	at org.python.pydev.ast.interpreter_managers.PythonInterpreterManager.internalCreateInterpreterInfo(PythonInterpreterManager.java:43)
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.createInterpreterInfo(AbstractInterpreterManager.java:329)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManager$CreateInterpreterInfoCallback.createInterpreterInfo(SyncSystemModulesManager.java:172)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManager.updateStructures(SyncSystemModulesManager.java:278)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler$SynchJob.run(SyncSystemModulesManagerScheduler.java:234)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:42:02.197
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:42:02.201
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:42:02.201
!MESSAGE Unable to find mypy. Project: sidh9846_data_structures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_data_structures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2023-11-18 17:42:02.440
!MESSAGE Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 50583

 - Environment:
APP_ICON_90082=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_90082=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.ubAWpVqq9T/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D3EC79B6-0B6D-40C8-ACCD-FC04C4262A55
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

!STACK 0
java.lang.RuntimeException: Error creating python process - exited before creating sockets - exitValue = (1).
ProcessInfo:

 - Executed: /usr/bin/python3 -u /Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pycompletionserver.py 50583

 - Environment:
APP_ICON_90082=../Resources/Eclipse.icns
HOME=/Users/manveer
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
JAVA_STARTED_ON_FIRST_THREAD_90082=1
LANG=en_CA.UTF-8
LOGNAME=manveer
MANPATH=/opt/homebrew/share/man::
OLDPWD=/Users/manveer
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.10/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
PWD=/Users/manveer
PYTHONPATH=/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc/pydev_sitecustomize:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload:/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
SHELL=/bin/zsh
SHLVL=1
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.ubAWpVqq9T/Listeners
TERM=xterm-256color
TERM_PROGRAM=Apple_Terminal
TERM_PROGRAM_VERSION=447
TERM_SESSION_ID=D3EC79B6-0B6D-40C8-ACCD-FC04C4262A55
TMPDIR=/var/folders/wq/3rym1bhx1fd7qszt7g2981d40000gn/T/
USER=manveer
XPC_FLAGS=0x0
XPC_SERVICE_NAME=0
_=/Users/manveer/eclipse/Eclipse.app/Contents/MacOS/eclipse
__CFBundleIdentifier=com.apple.Terminal
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x52

 - Working Dir:
/Users/manveer/eclipse/Eclipse.app/Contents/Eclipse/plugins/org.python.pydev.core_10.1.1.202301091814/pysrc

 - OS:
Mac OS X

 - Std output:


 - Err output:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

	at org.python.pydev.core.log.Log.log(Log.java:33)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.startIt(AbstractShell.java:287)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:240)
	at org.python.pydev.ast.codecompletion.shell.ShellsContainer.getServerShell(ShellsContainer.java:168)
	at org.python.pydev.ast.codecompletion.shell.AbstractShell.getServerShell(AbstractShell.java:219)
	at org.python.pydev.editor.PyEdit$2.run(PyEdit.java:560)
!SESSION 2024-03-18 18:13:59.202 -----------------------------------------------
eclipse.buildId=4.30.0.I20231201-0110
java.version=17.0.4.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_CA
Framework arguments:  -keyring /Users/manveer/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -data /Users/manveer/Documents/Eclipse/CP164 -keyring /Users/manveer/.eclipse_keyring

!ENTRY org.eclipse.jface 2 0 2024-03-18 18:14:17.326
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2024-03-18 18:14:17.326
!MESSAGE A conflict occurred for COMMAND+SHIFT+T:
Binding(COMMAND+SHIFT+T,
	ParameterizedCommand(Command(org.eclipse.jdt.ui.navigate.open.type,Open Type,
		Open a type in a Java editor,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		WorkbenchHandlerServiceHandler("org.eclipse.jdt.ui.navigate.open.type"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
Binding(COMMAND+SHIFT+T,
	ParameterizedCommand(Command(org.eclipse.lsp4e.symbolinworkspace,Go to Symbol in Workspace,
		,
		Category(org.eclipse.lsp4e.category,Language Servers,null,true),
		WorkbenchHandlerServiceHandler("org.eclipse.lsp4e.symbolinworkspace"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2024-03-18 18:14:17.326
!MESSAGE A conflict occurred for ALT+COMMAND+R:
Binding(ALT+COMMAND+R,
	ParameterizedCommand(Command(org.python.pydev.debug.setnext,Set Next Statement,
		,
		Category(org.python.pydev.ui.category.run,PyDev - Run,Python run category,true),
		WorkbenchHandlerServiceHandler("org.python.pydev.debug.setnext"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
Binding(ALT+COMMAND+R,
	ParameterizedCommand(Command(org.eclipse.jdt.ui.edit.text.java.rename.element,Rename - Refactoring ,
		Rename the selected element,
		Category(org.eclipse.jdt.ui.category.refactoring,Refactor - Java,Java Refactoring Actions,true),
		WorkbenchHandlerServiceHandler("org.eclipse.jdt.ui.edit.text.java.rename.element"),
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,cocoa,system)

!ENTRY org.python.pydev.shared_core 1 1 2024-03-18 18:14:22.366
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2024-03-18 18:14:22.369
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2024-03-18 18:14:22.375
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2024-03-18 18:14:22.379
!MESSAGE Unable to find mypy. Project: sidh9846_data_structures
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_data_structures
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 1 1 2024-03-18 18:14:22.379
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/lib-dynload
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 1 1 2024-03-18 18:14:22.380
!MESSAGE Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
!STACK 0
java.lang.RuntimeException: Unable to track file that does not exist: /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages
	at org.python.pydev.shared_core.log.Log.logInfo(Log.java:54)
	at org.python.pydev.shared_core.path_watch.PathWatch.track(PathWatch.java:325)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:459)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.afterSetInfos(SyncSystemModulesManagerScheduler.java:440)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.registerInterpreterManager(SyncSystemModulesManagerScheduler.java:66)
	at org.python.pydev.ast.codecompletion.revisited.SyncSystemModulesManagerScheduler.start(SyncSystemModulesManagerScheduler.java:88)
	at org.python.pydev.plugin.PydevPlugin$1.run(PydevPlugin.java:159)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2024-03-18 18:14:42.074
!MESSAGE Unable to find mypy. Project: sidh9846_a09
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_a09
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2024-03-18 18:15:17.796
!MESSAGE Unable to find pylint executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find pylint executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.pylint.PyLintPreferences.getPyLintLocation(PyLintPreferences.java:87)
	at com.python.pydev.analysis.pylint.PyLintVisitor.startVisit(PyLintVisitor.java:73)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2024-03-18 18:15:17.807
!MESSAGE Unable to find mypy executable. Searched in:
/usr/bin
!STACK 0
org.python.pydev.core.IInterpreterInfo$UnableToFindExecutableException: Unable to find mypy executable. Searched in:
/usr/bin
	at org.python.pydev.ast.interpreter_managers.InterpreterInfo.searchExecutableForInterpreter(InterpreterInfo.java:2069)
	at com.python.pydev.analysis.mypy.MypyPreferences.getMypyLocation(MypyPreferences.java:66)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:67)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2024-03-18 18:15:17.808
!MESSAGE Unable to find mypy. Project: sidh9846_a09
!STACK 0
java.lang.RuntimeException: Unable to find mypy. Project: sidh9846_a09
	at org.python.pydev.core.log.Log.log(Log.java:33)
	at com.python.pydev.analysis.mypy.MypyVisitor.startVisit(MypyVisitor.java:70)
	at com.python.pydev.analysis.additionalinfo.builders.AnalysisBuilderRunnable.doAnalysis(AnalysisBuilderRunnable.java:297)
	at com.python.pydev.analysis.additionalinfo.builders.AbstractAnalysisBuilderRunnable.run(AbstractAnalysisBuilderRunnable.java:164)
	at org.python.pydev.core.concurrency.RunnableAsJobsPoolThread$1.run(RunnableAsJobsPoolThread.java:140)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
